inc/class-upgrade.php

```diff
@@ -870,6 +871,23 @@ private function upgrade_183() {
 		$this->delete_post_meta( 'yoast-structured-data-blocks-images-cache' );
 	}
 
+	/**
+	 * Performs the upgrade routine for the first time configuration.
+	 *
+	 * @TODO Set the right version number when the feature branch is merged.
+	 */
+	private function upgrade_first_time_configuration() {
+		$other   = [];
+		$other[] = WPSEO_Options::get( 'instagram_url' );
```

Comment:
We probably need to check if there _is_ an instagram_url before we import it to the new setting. The same goes for the rest of the options there.

If we don't, we will always have 6 array elements there and all of them might be empty strings